src/burn/stub/StubSection.cpp

```diff
@@ -18,6 +18,5 @@ static DWORD dwOriginalSignatureSize = 0;
 
 static DWORD dwContainerFormat = 1;
 static DWORD dwContainerCount = 0;
-static DWORD qwBootstrapperApplicationContainerSize = 0;
-static DWORD qwAttachedContainerSize = 0;
+static DWORD qwAttachedContainerSizes[116]; // Including UX container
```

Comment:
Probably should have comment explaining the math to get to 512 bytes, the minimum size of a section.